src/esssans/io.py

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2023 Scipp contributors (https://github.com/scipp)
+
+import scipp as sc
+import scippnexus as snx
+from scippnexus.application_definitions import nxcansas
+
+from .types import BackgroundSubtractedIofQ, OutFilename, RunNumber, RunTitle
+
+
+def save_background_subtracted_iofq(
+    *,
+    iofq: BackgroundSubtractedIofQ,
```

Comment:
Do we ever need to store the result without background subtraction?

Reply:
Don't know. Given how Sciline works, we would need two separate functions. We can easily add another one if we need to.